learning-progression-with-la-schema.md

```diff
@@ -71,7 +71,7 @@ The way MakeCode exposes libraries like [`game.ts`](https://github.com/microsoft
 
```

Comment:
For some reason I can't leave a comment on line 68.
When you say, "A runtime is a set of programs which translates human-readable programs in a particular programming language into a lower layer of software stack, which is usually much closer to the particular physical device that will run the programs." 
Do you mean that the lower level code is more specific to the physical device?

Reply:
Yes. It's quite bewildering this way. I will do a pass through that sentence.